### PR TITLE
updating alias

### PR DESCRIPTION
**Pull request by harshulgupt**
### :pencil: Description

**Type:** :beetle: `bugfix` | :rocket: `feature` | :biohazard: `breaking change` | :vertical_traffic_light: `testing` | :memo: `documentation` | :roller_coaster: `infrastructure`

Write a complete description of your changes, including the necessary context or any piece of information required to understand your work.

Also, link issues affected by this pull request by using the keywords: `close`, `closes`, `closed`, `fix`, `fixes`, `fixed`, `resolve`, `resolves` or `resolved`. 

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: content/gsoc_2023/gsoc_home.md [hunk 1]
```diff
@@ -58,7 +59,8 @@ Since 2005, Google has run an awesome open source software development program,
   - That, in turn, determines how many student developers work with the TARDIS collaboration.
 
 Please also read the following pages before applying. Project ideas can be found on the [ideas page](../ideas).
+
 1. [Application Requirements](../requirements)
 2. [Application Guidelines](../guidelines)
 3. [Application Checklist](../checklist)
-5. [Participation Expectations](../expectations)
+4. [Participation Expectations](../expectations)
```
Comment (atharva-2001) [inline content/gsoc_2023/gsoc_home.md:66]: These aren't relevant to the pull request, right?

File: content/research/papers.md [hunk 2]
```diff
@@ -3,153 +3,107 @@ title: "Papers"
 date: 2021-05-28T09:27:27-05:00
 draft: false
 layout: page
+aliases: /research/
 ---
-## Papers using the *tardis* package
 
-**Dutta, Anirban, Singh, Avinash, Anupama, G. C., et al.** 2021, MNRAS, *"SN 2017hpa: a carbon-rich Type Ia supernova"* <a href="https://ui.adsabs.harvard.edu/abs/2021MNRAS.503..896D" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+## Papers using the _tardis_ package
```
Comment (atharva-2001) [inline content/research/papers.md:10]: I'm afraid I don't quite understand why this change(and multiple following) was done. Did you run some sort of formatter?

**Review by atharva-2001 — CHANGES_REQUESTED** [review]

**Comment by atharva-2001** [comment]
Thank you for creating this pull request @harshulgupt! I made a few comments, please look at them at your convenience. Also, is there a preview link?

File: content/research/papers.md (hunk 2 above)
Comment (harshulgupt) [reply content/research/papers.md:10]: My VS Code is set to format on save. Maybe that is why it happened.

File: content/gsoc_2023/gsoc_home.md (hunk 1 above)
Comment (harshulgupt) [reply content/gsoc_2023/gsoc_home.md:66]: No I don't think so. I don't remember making thesw changes. Only aliases. And removing to .md files which were not necessary

**Comment by atharva-2001** [comment]
@harshulgupt Please let me know if you need any help with this.

File: .github/workflows/website-preview.yml [hunk 3]
```diff
@@ -2,6 +2,7 @@ name: Preview website on fork
 
 on:
   push:
+  workflow_dispatch:
```
Comment (atharva-2001) [inline .github/workflows/website-preview.yml:6]: @harshulgupt I don't think we need to change this. It just provides an option to run the workflow manually.

Comment (atharva-2001) [reply .github/workflows/website-preview.yml:6]: I see!

Comment (atharva-2001) [reply .github/workflows/website-preview.yml:6]: Just a note- the website preview workflow runs every time we make changes.
